parse-config.sh

```diff
@@ -29,49 +29,35 @@ load_dir() {
 
         # $3, is the numbers of characters to strip in front of the key
         KEY="${KEY:$3}"
+        
+        # Removes the application.$application_name prefix if it exists
+        if [[ "$KEY" == "application.$APPLICATION_NAME."* ]]; then
+          KEY=${KEY#"application.$APPLICATION_NAME."}
+        fi
+
+        # Removes the .terraform suffix if it exists
+        if [[ "$KEY" == *".terraform" ]]; then
+          KEY=${KEY%".terraform"}
+        fi
```

Comment:
Actually, I think this we don't need anymore, right? 🤔 And existing properties don't have it if I remember correctly since they weren't terraformed, but we should double check

Reply:
yeah, not sure. Let's remove with all the others. I will add TODOs